src/review/logic.py

```diff
@@ -77,6 +81,31 @@ def get_reviewers(article, candidate_queryset, exclude_pks):
     ).annotate(
         rating_average=Subquery(rating_average, output_field=IntegerField()),
     )
+
+    if article.journal.get_setting('general', 'display_past_reviewers'):
+        completed_reviewer_pks = article.completed_reviews_with_decision.values_list(
+            'reviewer__pk',
+            flat=True,
+        )
+        reviewers = reviewers.annotate(
+            is_past_reviewer=Case(
+                When(pk__in=completed_reviewer_pks, then=True),
+                default=False,
```

Comment:
We could pass in a subquery or do an inner join to save us one roundtrip to the database but I'm not quite sure what the performance would be like, what have you found locally with this implementation?